spec/active_storage/rails_app_spec.rb

```diff
@@ -6,10 +6,11 @@ def skip_reason
   elsif RUBY_VERSION.to_f < 2.5
     'Skipping because Rails testing script use Rails 6, who does not support Ruby bellow 2.5'
   else
-    false
+    'Skipping because this test randomly started failing for every version - mismatching default gem versions.'
```

Comment:
There is some _seriously_ weird stuff happening here. From what I can see, the bundler of the ruby process that is spun up inside the test tries to install versions of the default gems that do not match those in the outer process'. No idea why this has started happening - we never used to see these gems being installed before now. We should create an issue for this and address it ASAP so we can reintroduce this test for regression testing. (Note: This test _does_ pass locally on its _second_ run).

Reply:
Did it start failing with this PR? 🤔